### PR TITLE
landing page

### PR DESCRIPTION
**Pull request by hiepvg1999**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/39842

## WHAT
- Create landing page

## HOW
- N/A

## WHY
- N/A

## Evidence (Screenshot or Video)
image
image
image

### CODE REVIEW

**Comment by hiepvg1999** [comment]
ready

File: hotel/templates/landing_page/page.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{% load i18n %}
+{% load static %}
+<img class="image" src="{% static 'image/common/hotel.jpg' %}" alt="The Hotel" style="width:100%;">
```
Comment (huongnt-2545) [inline hotel/templates/landing_page/page.html:3]: translate text "The Hotel"

**Review by huongnt-2545 — APPROVED** [review]
merged
Chị có 1 comment nhỏ thôi, em fix ở pull sau nhé

File: hotel/templates/landing_page/page.html (hunk 1 above)
Comment (hiepvg1999) [reply hotel/templates/landing_page/page.html:3]: oke chị

**Pull request merged**